pkg/controlplane/control/manager.go

```diff
@@ -243,31 +243,40 @@ func (m *Manager) AddImport(ctx context.Context, imp *v1alpha1.Import) (err erro
 		return err
 	}
 
-	if imp.Namespace == m.namespace {
-		return nil
+	if imp.Namespace != m.namespace {
+		userService := &v1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      imp.Name,
+				Namespace: imp.Namespace,
+				Labels:    make(map[string]string),
+			},
+			Spec: v1.ServiceSpec{
+				ExternalName: fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, m.namespace),
+				Type:         v1.ServiceTypeExternalName,
+			},
+		}
+
+		if err := m.addImportService(ctx, imp, userService); err != nil {
+			return err
+		}
 	}
 
-	userService := &v1.Service{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      imp.Name,
-			Namespace: imp.Namespace,
-			Labels:    make(map[string]string),
-		},
-		Spec: v1.ServiceSpec{
-			ExternalName: fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, m.namespace),
-			Type:         v1.ServiceTypeExternalName,
-		},
+	if imp.Spec.Alias != "" {
+		if err := addCoreDnsRewrite(ctx, m, &importName, imp.Spec.Alias); err != nil {
+			m.logger.Errorf("Failed to configure CoreDns: %v.", err)
```

Comment:
```suggestion
			m.logger.Errorf("failed to configure CoreDNS: %v.", err)
```

Reply:
Also, if `logger.Errorf` supports `%w`  - prefer that over `%v`

Reply:
I get compiler warning: `(*github.com/sirupsen/logrus.Entry).Errorf does not support error-wrapping directive %w`.
I suggest for now to leave `%v`.
Thanks